app/features-json/setting_json_controller.rb

```diff
@@ -0,0 +1,29 @@
+require_relative "api_controller"
+
+module FastlaneCI
+  # Controller for providing all data relating to system settings
+  class SettingJSONController < APIController
+    HOME = "/data/settings"
+
+    get HOME do
+      return json(Services.setting_service.settings)
+    end
+
+    post HOME do
+      metrics_setting = Services.setting_service.find_setting(setting_key: :metrics_enabled)
+      metrics_setting.value = false
+      Services.setting_service.update_setting!(setting: metrics_setting)
+
+      return json({ status: :success })
```

Comment:
how would we determine if this failed? Would we need to catch an exception or something?

Reply:
Yah, right now we have the `verify_block` which could potentially raise an exception, so the caller of `setting_name.value = something` has to handle this